docs/admin/authentication.md

```diff
@@ -33,7 +33,9 @@ plane, must authenticate when making requests to the the API server.
 ## Authentication strategies
 
 Kubernetes uses client certificates, bearer tokens, or HTTP basic auth to
-authenticate API requests through authentication plugins. As HTTP request are
+authenticate API requests through authentication plugins. A bearer token
+must be something that can be included in an HTTP header value without any
```

Comment:
I agree that a concrete example is good.  I think ideal doc has both a general statement and one or more completely concrete examples.

Since what is true about the syntactic considerations for bearer tokens is true of all four cases, I would prefer to see it written down once up front rather than repeated four times, or various fragments repeated in different cases.